### ISSUE TITLE
Add missing accessibility description to p5.Color._parseInputs()

### ISSUE THREAD

**Issue by Zearin** [processing/p5.js#5584]
### Most appropriate sub-area of p5.js?

- [X] Accessibility (Web Accessibility)
- [ ] Build tools and processes
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Friendly error system
- [ ] Image
- [ ] IO (Input/Output)
- [ ] Localization
- [ ] Math
- [ ] Unit Testing
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

### p5.js version

1.4.0

### Steps to reproduce this

[I was asked](https://github.com/processing/p5.js/pull/5577#discussion_r795217753) by @lm-n to open an Issue regarding a missing accessibility description to `p5.Color._parseInputs()`.

Even though it’s a private function, I asked @lm-n if it still needed a `describe()`. His response:

> doesn't matter for reference but could be helpful for contributors to see an example

Which works well enough for me! 👍

**Comment by Qianqianye** [comment]
Hi @nickmcintyre, just checking in with you to see if we have this resolved in our recent effort of updating p5.js reference page. Thank you!

**Comment by nickmcintyre** [comment]
@Qianqianye thanks for including me! This issue wasn't resolved in the first batch of edits. I don't think this method is actually "missing" an accessibility description. It isn't part of the public API and is only called internally.

As for adding an example to the reference, @paulaxisabel, @SoundaryaKoutharapu, @mrbrack, @TJ723, @Zarkv, @SkylerW99, @ramya202000, @hannahvy, @robin-haxx, @hiddenenigma I think y'all are better positioned to weigh in.

**Comment by limzykenneth** [comment]
It will just be an internal reference so I think adding an example won't be necessary for this case?
